### ISSUE TITLE
Command history can't switch using arrow keys

### ISSUE THREAD

**Issue by codejamtc** [topcoderinc/TC-redis-manager#313]
![image](https://user-images.githubusercontent.com/42398485/51069284-7114c900-15f1-11e9-86d4-c131f1be8660.png)

1. Deploy the App and run the application
2. Select a server and click on the redis command cli
3. Use some commands
4. Click on the command text field and click up down arrows

**Actual:** Command history can't switch using arrow keys

**Expected:** Should be able to switch between used commands using arrow keys

**Browser(s):** Firefox 64.0.2 (Windows)

**Comment by standlove** [comment]
are you using the latest commit? @codejamtc

**Comment by codejamtc** [comment]
@standlove yes I have used the latest code and I can see the latest commit is 2 days ago

**Comment by standlove** [comment]
Our latest commit should have fixed this issue ... I cannot reproduce this ...
